ui/src/notebooks/pipes/Query/view.tsx

```diff
@@ -40,10 +47,16 @@ const Query: FC<PipeProp> = ({data, onUpdate, Context, results}) => {
           onSubmitScript={() => {}}
           autogrow
         />
-        <Results results={results} size={size} onUpdateSize={onUpdateSize} />
+        <Results
+          results={results}
+          height={resultsHeight}
+          onUpdateHeight={handleUpdateResultsHeight}
```

Comment:
why isn't height, onUpdateHeight, visibility, and onUpdateVisibility self contained within the results component? it's strange to route the data in and out like that instead of just routing the meta data into the results component

Reply:
Should results just accept `data` and `onUpdate` as props instead of breaking them down first?